src/core/trulens/core/app/custom.py

```diff
@@ -3,15 +3,15 @@
 
 This wrapper is the most flexible option for instrumenting an application, and can be used to instrument any custom python class.
 
-!!! example
+Example:
```

Comment:
I thought doing !!!example, will put this in a formatted box for the docs.

Reply:
[Either will way will show admonitions on the docs](https://mkdocstrings.github.io/python/usage/docstrings/google/?h=admonition#docstring). Figured this might be more readable as a docstring.